modules/ROOT/pages/to-download-and-install-studio-wx.adoc

```diff
@@ -8,11 +8,25 @@ endif::[]
 . Download the Anypoint Studio installation file from the https://www.mulesoft.com/lp/dl/studio[Download Site].
 . Extract the Windows installation file into the `C:\` root folder.
 ** Ensure that your user has read and write permissions for the `C:\` directory.
-** If you are using a Windows antivirus application, ensure that the `plugins/` and `features/` directories are under the _trusted_ category.
+** If you are using a Windows antivirus application, ensure that the `plugins/` and `features/` directories are under the *trusted* category.
 . Open the extracted Anypoint Studio.
 +
 When Studio first runs, the embedded OpenJDK Platform libraries might cause a Windows Security alert. +
 Ensure to allow the OpenJDK Platform library from Temurin publisher through your Windows firewall.
-. Click OK to accept the default workspace. +
+. Click *OK* to accept the default workspace. +
 ** If you choose to use an existing workspace, this must have been created using Anypoint Studio 7.0 or later. +
 Studio 7.x supports only Mule 4.x projects, because the structure of the project, export format, XML, and scripting language are different than the ones used by Mule 3.x.
+
+== Exclude Studio from Windows Defender Scans
+
+While Windows Defender is a tool for keeping your computer safe, it can also slow down the performance of Studio because it scans files in real-time as they are accessed, which causes delays. To prevent performance issues, you can exclude Studio from Windows Defender scans. 
+
+. Run Studio. The following window pop-ups to ask if Studio folders and processes should be excluded:
```

Comment:
After running Studio_, if Windows Defender Real Time Scanning is enabled,_ the following window pop-ups to ask if Studio folders and processes should be excluded. 

Windows Defender is not the only real time scanning virus tool the user can has, and this feature only works with Windows Defender. i.e. Salesforce machines has Falcon, so this feature doesn't work for that windows machines.

Reply:
Thanks for the clarification! 
It may be misleading for the reader if we don't mention "Windows Defender" in the first step.

Reply:
Made some changes! 
Let's see if they work! :)